app/models/post.rb

```diff
@@ -0,0 +1,4 @@
+class Post < ApplicationRecord
+  has_many :comments, dependent: :destroy
+  belongs_to :blog
+end
```

Comment:
You mentioned the posts being "text only" requirement. This probably needs clarification in the readme. What we mean is that posts should only consist of a body of text, and that additional features like image posts, or videos, etc, would be one of the 'extra features'.

However, if you did want to limit the body content of a post, you'd do it with validations. For example, something like this might work, in your Post class:

```ruby
# exclude special characters from body content
validates :body, format: {
    with: /\A[a-zA-Z\d ]+\z/,
    message: "only allows letters" }
```

There are all sorts of types of validations for making sure data that gets into the database is proper. These validations will also show error messages on the form. 

(I notice that your Post form actually uses `@blog.posts.build` as the second model param, meaning it's actually building a new post every time the form renders, so the form will appear empty if it fails. Instead, you'd probably want to do something like

```
<%= simple_form_for([@blog, @post || @blog.posts.build]) do |f| %>
```

Or, you could build a `@post` instance variable in any controller action methods that need it, and just do

```
<%= simple_form_for([@blog, @post]) do |f| %>
```

If that makes any sense!

Reply:
I think it does, but I'll read through it again and try things out and see how I go! The second code you suggest is something I came across a lot when searching online, and now with your comments on I am going to have another look at it.

Re: the 'text-only' requirement: you know now that you mention its just text and not images etc, it makes perfect sense! For some reason when I saw that I took it literally (in the formal sense).  Thanks for the code snippet, I like that,! I came across validations when looking at my tests, so I'll play around with them more :)